Makefile

```diff
@@ -29,11 +29,11 @@
 GPP := g++
 GPP_FLAGS := -Wall -Wmain -pedantic -pedantic-errors -std=c++11
 INCLUDE_FLAGS := \
-    -I. -Iexternal -Iexternal/dimensional_types -Iexternal/progress_bar
+    -I. -Iglad/include -Iexternal -Iexternal/dimensional_types -Iexternal/progress_bar
```

Comment:
Can you move the glad files to external using
  external/glad/include/glad.h
  extenal/glad/src/glad.cc
  extenal/glad/regenerate.sh